docs/docs/HowTo/schemas/Approve-a-schema.md

```diff
@@ -0,0 +1,40 @@
+# Manage schema
+
+When a schema request is made, a team member other than the request
+creator can review, approve, or decline the request. Schema requests can
+only be submitted by the topic owner team and approved by the same team.
+
+## Approve a schema request
+
+1.  When you login into Klaw, a pop-up will notify you of **Pending
+    Requests**. Select **Yes, show me!** or navigate to **Approve** and
+    select **Schema Requests** to view the pending requests.
+2.  Review all information related to the request and approve it by
+    selecting the tick button while viewing the pending schema requests.
+3.  The schema will immediately attempt to be provisioned on approval.
+
+> If there is any issue with the schema creation, the request will remain
```

Comment:
Another instance of '>' rendering incorrectly.

![image](https://github.com/aiven/klaw-docs/assets/121855584/60230e20-bc0f-4745-9bff-f8d6cb39a545)